src/actions/NekInitAction.C

```diff
@@ -156,6 +155,11 @@ NekInitAction::act()
 
   _n_cases++;
 
+  if (nekrs::hasMovingMesh() && _type != "NekRSProblem")
```

Comment:
I think this check is a bit narrower than we need - technically, `NekRSStandaloneProblem` should allow any valid NekRS input file to run within Cardinal (it doesn't have any data transfers to/from). So I think we want this to swap out `_type != "NekRSProblem"` with `_type == "NekRSSeparateDomainProblem"`

Reply:
After this change is made, I'd suggest adding a `NekRSStandaloneProblem` test that just runs the `mv_cyl` example

Reply:
Done, thanks! Waiting for tests to pass...